### ISSUE TITLE
10 Ingredients in Weight Loss Pills

### ISSUE THREAD

**Issue by nadjibandnaaima** [nadjibandnaaima/diets-for-quick-weight-loss#15]
<div dir="ltr" style="text-align: left;">
<h2 style="text-align: left;">10 Ingredients in Weight Loss Pills</h2>
<h2 style="text-align: left;">10 Ingredients in Weight Loss Pills</h2>
<br><b>10 Ingredients in Weight Loss Pills</b><br><br><b>
<ins class="adsbygoogle" data-ad-client="ca-pub-2862230082201059" data-ad-slot="1546920322" style="display: inline-block; height: 600px; width: 300px;"></ins> 
//<![CDATA[
(adsbygoogle = window.adsbygoogle || []).push({});
//]]>
</b><br><br>
Looking for help with losing weight? Help that goes beyond the usual lifestyle changes, portion control advice and exercise routines? Weight-loss supplements touting names like "fat burner" and "thermo max" can be enticing. But do they work? And even more important, in the wake of the Food and Drug Administration's recent ban on ephedra, are they safe?<br><br>
In this article, Environmental Nutrition provides a snapshot of 10 of the most popular ingredients currently in supplements for weight loss.<br><br>
Chitosan<br>
Made from the shells of lobsters, shrimp and other shellfish, chitosan is an indigestible fiber. It is supposed to help weight loss by binding with the fat you eat, blocking its absorption. However, studies do not support this claim. Side effects include constipation and gas. People with shellfish allergies should beware.<br><br>
Chromium<br>
Most often present as chromium picolinate, it has been promoted to boost lean mass and decrease body fat. Though study findings have been inconclusive, an analysis of several studies recently concluded that supplementing with 200 to 400 micrograms of chromium picolinate could result in about a 2.5-pound weight loss per week in some people. Other studies have found a loss of body fat and an increase in muscle tissue, with no change in weight. Still others have found no effect at all. Though some laboratory studies have raised safety issues, a large number of human studies suggest it is safe.<br><br>
Citrus Aurantium (Bitter Orange)<br>
This herbal extract is a natural source of synephrine (a compound similar to ephedrine) and the supplement industry's current answer to the ephedra ban. But there are no published studies to show it is an effective supplement for weight loss. And like ephedra, its side effects include an increase in blood pressure; Environmental Nutrition recommends avoiding it.<br><br>
Conjugated Linoleic Acid (CLA)<br>
Both animal and human research hints at a possible role for CLA in reducing body fat and increasing lean body mass, but it's not quite ready for prime time. None of the human studies have been long-term. CLA is currently being studied in combination with chromium picolinate. CLA is found naturally in dairy foods and beef and is generally considered safe.<br><br>
Garcinia Cambogia<br>
The extract from Garcinia cambogia fruit rind contains hydroxycitric acid (HCA), which supposedly curbs food intake, weight gain and storage of excess calories as fat. However, studies have failed to prove G. cambogia is an effective weight-loss aid. The most common side effects are upper respiratory and gastrointestinal symptoms.<br><br>
Guarana<br>
Caffeine is the effective ingredient in guarana plant seeds. Guarana may have a mild appetite suppressant effect over the short term. It was often used in combination with ephedra until ephedra was banned. The potential side effects are those of caffeine overdose&mdash;insomnia, restlessness, anxiety or panic.<br><br>
Hydroxymethylbutyrate (HMB)<br>
HMB is a natural byproduct of the breakdown of the amino acid leucine. HMB supplements are used by body builders in hopes of increasing muscle mass. A few small studies have suggested that HMB increases lean mass and decreases body fat when combined with weight training. However, studies with overweight and obese people are needed.<br><br>
Plantago Psyllium<br>
Psyllium is a water-soluble fiber from Plantago ovato seeds. Taken before a meal, it may make you feel full, causing you to eat less. While it is proved to aid constipation (that's how Metamucil works), it has not been proved as an effective weight-loss aid.<br><br>
Pyruvate<br>
Pyruvate formulas boast that they stimulate metabolism and reduce body fat. Pyruvate is found naturally in food and in the body as a result of the breakdown of carbohydrates and protein. Some studies have shown a weight-loss benefit with pyruvate supplements. But the people in the studies also exercised and cut back on calories. Moreover, the dosage used in the studies was high, requiring a handful of tablets each day. Experts say the case for pyruvate to aid weight loss is weak.<br><br>
Yerba mate<br>
Brewed into a tea, bitter Yerba mate is high in caffeine. Used in combination with other caffeine-containing herbs, like guarana and damiana, it was found in one study to delay stomach emptying time and aid weight loss. The study was short (45 days), so conclusions on keeping the weight off cannot be made.<br><br>
The Bottom Line<br><br>
Though all are relatively expensive and none are proved, of these top popular weight-loss contenders, chromium picolinate and CLA are probably the most studied. If you've already made up your mind to go the supplement route, you might want to give them a try. Just don't expect miracles and check with your doctor first.<br><br><br><br><br><br>
</div>
<br><br>
via diets for quick weight loss http://ift.tt/1KHzwKw<br>
September 29, 2015 at 10:51AM
